src/docker/workflows/cp-schema-registry.ts

```diff
@@ -0,0 +1,386 @@
+import { CancellationToken, commands, Progress, window } from "vscode";
+import { getKafkaWorkflow } from ".";
```

Comment:
This introduces a circular dependency, but no issues with build/run -- will be cleaned up in a follow-on branch for https://github.com/confluentinc/vscode/issues/530